packages/google_adsense/lib/src/ad_unit_widget_web.dart

```diff
@@ -68,7 +62,11 @@ class AdUnitWidgetWeb extends AdUnitWidget {
   final Map<String, String> _unitParams;
 
   final web.HTMLElement _insElement =
-      web.document.createElement('ins') as web.HTMLElement;
+      (web.document.createElement('ins') as web.HTMLElement)
+        ..className = 'adsbygoogle'
+        ..style.display = 'block'
+        ..style.width = '100%'
```

Comment:
When using 100% width the ad overflows, while without setting it to 100% it takes all the available space of parent properly anyways. Not sure if I'm missing anything here

Reply:
Woops, I probably left this while I was testing other stuff. Feel free to remove, I don't remember at what point I needed this!

Reply:
Ah great! Thanks for confirming!